clients/lsp-steep.el

```diff
@@ -47,12 +47,17 @@
   :safe #'booleanp
   :group 'lsp-steep)
 
+(defcustom lsp-steep-server-path nil
+  "Path of the Steep language server executable.
+If specified, `lsp-steep-use-bundler' is ignored."
+  :type 'file
```

Comment:
can you add version 7.1 as we do for the other properties?

Reply:
Sure. 👍🏼 
Is it desirable to add `:package-version \`(lsp-mode . "7.1.0")` also to the existing variables `lsp-steep-log-level` and `lsp-steep-use-bundler`?

Reply:
Done 638c0200. It seems good to add 7.1 only to the new variable.